README.rst

```diff
@@ -29,9 +29,9 @@ a. Check requirements
 
 Make sure that you have installed `Python <https://www.python.org/>`_ (>=
 3.4) on your system. You will also need the Python development headers (e.
-g. ``python3-dev`` for Ubuntu).
-
-TODO: Check Ubuntu 14.04 and extra packages like libjpeg ...
+g. ``python3-dev`` for Ubuntu), the Independent JPEG Group's JPEG runtime
+library (dependency package) (e. g. ``libjpeg-dev`` for Ubuntu) and the
+compression library (development) (e. g. ``zlib1g-dev`` for Ubuntu).
 
```

Comment:
please add a copy-paste-command for debian/ubuntu:
`apt-get install python3-dev libjpg-dev zlib1g-dev`